etc/profile-m-z/psi.profile

```diff
@@ -0,0 +1,78 @@
+# Firejail profile for psi
+# Description: Native XMPP client with GPG support
+# This file is overwritten after every install/update
+# Persistent local customizations
+include psi.local
+# Persistent global definitions
+include globals.local
+
+# Uncomment for GPG
+# noblacklist ${HOME}/.gnupg
+noblacklist ${HOME}/.cache/psi
+noblacklist ${HOME}/.cache/Psi
+noblacklist ${HOME}/.config/psi
+noblacklist ${HOME}/.local/share/psi
+noblacklist ${HOME}/.local/share/Psi
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+# Uncomment for GPG
+# mkdir ${HOME}/.gnupg
+mkdir ${HOME}/.cache/psi
+mkdir ${HOME}/.cache/Psi
+mkdir ${HOME}/.config/psi
+mkdir ${HOME}/.local/share/psi
+mkdir ${HOME}/.local/share/Psi
+# Uncomment for GPG
+# whitelist ${HOME}/.gnupg
+whitelist ${HOME}/.cache/psi
+whitelist ${HOME}/.cache/Psi
+whitelist ${HOME}/.config/psi
+whitelist ${HOME}/.local/share/psi
+whitelist ${HOME}/.local/share/Psi
+whitelist ${DOWNLOADS}
+# Uncomment for GPG
+# whitelist /usr/share/gnupg
+# whitelist /usr/share/gnupg2
+whitelist /usr/share/psi
+# Uncomment for GPG
+# whitelist ${RUNUSER}/gnupg
+# whitelist ${RUNUSER}/keyring
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+novideo
+nou2f
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+shell none
+# breaks on Arch
+# tracelog
+
+disable-mnt
+# Add "gpg,gpg2,gpg-agent,pinentry,pinentry-gtk-2" for GPG
```

Comment:
Depending on DE other pinentry binaries are needed. Is pinentry for all distros a symlink? if so, the default binary is copied.

Reply:
Not a symlink for me on Arch...

Reply:
Can you add more pinentry binaries.

Reply:
```
pinentry-curses
pinentry-fltk
pinentry-gnome3
pinentry-gtk2 or pinentry-gtk-2
pinentry-qt
pinentry-qt4
pinentry-tty
pinentry-x2go
pinentry-kwallet 
```

Reply:
`pinentry-gtk` and `pinentry-emacs`(?)

Reply:
I don't have a gtk; emacs one I forgot...